frontend/src/common/colors.js

```diff
@@ -0,0 +1,11 @@
+
+const tintColor = 'red';
+const textColor = '#333';
+
+export default {
+  textColor,
+  tintColor,
+  lightenGrey: '#f5f5f5',
```

Comment:
why are you saving tintColor in a const above, but export tealColor here this way, and not savin it in a const? if the design changes, all the colors will change. It doesn't look nice this way, can you please explain your reasons behind this?

Reply:
if I use this,
```js
export const tintColor = 'red';
export const textColor = '#333';
```
I can import it by the way blow
```js
import { tintColor } from '../common/colors';
```
but I changed my mind and I think this way would be better.
we will be consequent from now on in our group.
```js
export default {
  tintColor: 'red',
  textColor: '#333'
}

import Colors from '../common/colors';

const styles = {
  color: Colors.tintColor
}
```